*** c:\progra~1\Apache Group\Apache\src\os\win32\os.h	Mon Sep 13 08:32:42 1999
--- os.h	Fri Jul 21 08:20:20 2000
***************
*** 68,74 ****
--- 68,78 ----
  
  #include <windows.h>
  #include <winsock2.h>
+ 
+ #ifndef __MINGW32__
  #include <mswsock.h>
+ #endif
+ 
  #include <process.h>
  #include <malloc.h>
  #include <io.h>
***************
*** 108,115 ****
--- 112,123 ----
  #define HAVE_DRIVE_LETTERS
  typedef int uid_t;
  typedef int gid_t;
+ 
+ #ifndef __MINGW32__
  typedef int pid_t;
  typedef int mode_t;
+ #endif
+ 
  typedef char * caddr_t;
  
  /*
***************
*** 132,138 ****
--- 140,148 ----
  #define strncasecmp(s1, s2, n) strnicmp(s1, s2, n)
  #define lstat(x, y) stat(x, y)
  #define S_ISLNK(m) (0)
+ #ifndef __MINGW32__
  #define S_ISREG(m) ((m & _S_IFREG) == _S_IFREG)
+ #endif
  #ifndef S_ISDIR
  #define S_ISDIR(m) (((m) & S_IFDIR) == S_IFDIR)
  #endif
